.pre-commit-config.yaml

```diff
@@ -16,9 +16,11 @@ repos:
         args:
           - --exit-non-zero-on-fix
           - --config=pyproject.toml
+        exclude: "tutorials/"
       - id: ruff-format
         args:
           - --config=pyproject.toml
+        exclude: "tutorials/"
```

Comment:
**suggestion:** better to add this excludes to the ruff definition in `pyproject.toml` so the configuration will apply if someone just runs ruff directly from the cli too